docs/DeveloperGuide.md

```diff
@@ -842,8 +851,6 @@ Use case ends.
 
       Use case resumes at step 1.
 
-*{More to be added}*
-
 ### Non-Functional Requirements
```

Comment:
If you have not, I'd suggest to review this section as well. If you have done this and found that no changes are needed, ignore this comment.